File: TsinghuaNet.UWP/Contents/IUserContent.vb
Public Interface IUserContent
    Property User As FluxUser

    Property OnlineTime As TimeSpan

    Property IsProgressActive As Boolean

    Sub BeginAnimation()

    Function AddOneSecond() As Boolean
End Interface

Module UserContentHelper
    Public Function Max(d1 As Double, d2 As Double) As Double
        Return Math.Max(d1, d2)
    End Function
End Module

File: TsinghuaNet.Uno/TsinghuaNet.Uno.Shared/Contents/IUserContent.cs
using System;

namespace TsinghuaNet.Uno.Contents
{
    public interface IUserContent
    {
        FluxUser User { get; set; }
        TimeSpan OnlineTime { get; set; }
        bool IsProgressActive { get; set; }
        void BeginAnimation();
        bool AddOneSecond();
    }

    static class UserContentHelper
    {
        public static double Max(double d1, double d2) => Math.Max(d1, d2);
    }
}
